app/views/images/index.html.haml

```diff
@@ -0,0 +1,24 @@
+- if @images.any?
+  - @images.each do |image|
+    %h2= image.name
+    = image_tag image.image.url(:thumb)
+    %br
+    = link_to image.name, image_path(image)
+    - if current_user == image.user
+      = link_to "Edit image", edit_image_path(image)
+      = link_to "Delete image", image_path(image), method: :delete
+    %br
+    %span.likes_count= image.likes.count
+    = 'like'.pluralize image.likes.count
+    = link_to "Like", image_likes_path(image), class: 'likes-link'
+    %br
+    = link_to "Add comment", new_image_comment_path(image)
+    - if image.comments.any?
+      %ul
+        - image.comments.each do |comment|
+          %li
+            = comment.comments
+- else
+  No images yet
+%br
+%a(href='images/new') Add an image
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))